lib/simple-dom/html-serializer.js

```diff
@@ -59,7 +59,8 @@ HTMLSerializer.prototype.escapeText = function(textNodeValue) {
 HTMLSerializer.prototype.text = function(text) {
   var parentNode = text.parentNode;
   if(parentNode && (parentNode.nodeName === "STYLE" ||
-    parentNode.nodeName === "SCRIPT")) {
+    parentNode.nodeName === "SCRIPT" ||
+    parentNode.nodeName === "CODE")) {
```

Comment:
I am not sure this is the correct fix. SCRIPT and STYLE tags are special because of the actual languages: JavaScript allows `1 > 2` and CSS allows `.foo > .bar` where the brackets (`<` and `>`) have meaning in the language. This is not the case with `<code/>` blocks where the contents are still just plain HTML. What is the use case where you do not want the content encoded? Consider this in your browser console:

```js
var UNENCODED = "1 < 2";

var code = document.createElement("code");
code.appendChild(document.createTextNode(UNENCODED));
document.body.appendChild(code);
console.assert(code.innerHTML === UNENCODED, 'CODE blocks are encoded'); // it's encoded

var script = document.createElement("script");
script.appendChild(document.createTextNode(UNENCODED));
document.body.appendChild(script);
console.assert(script.innerHTML === UNENCODED, 'SCRIPT blocks are not encoded');
```